File: TDD Boot Camp/Part 01/Code/OOPrinciples/src/com/oop/abstraction/Employee.java
package com.oop.abstraction;

public class Employee
{
	private String name;
	private String id;
	private double salary;
	private String departmentName;
	public Employee(String name, String id, double salary,String departmentName)
	{
		this.name = name;
		this.id = id;
		this.salary = salary;
		this.departmentName = departmentName;
	}
	public String getName()
	{
		return name;
	}
	public String getId()
	{
		return id;
	}
	public Double getSalary()
	{
		return salary;
	}
	public String getDepartment() {
		return departmentName;
	}
}

File: TDD Boot Camp/Part 01/Code/C#/OOPrinciples/Abstraction/Employee.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Abstraction
{
    public class Employee
    {
        public string Name { get; set; }
        public string Id { get; set; }
        public double Salary { get; set; }
        public string DepartmentName { get; set; }
        public Employee(string name, string id, double salary, string departmentName)
        {
            Name = name;
            Id = id;
            Salary = salary;
            DepartmentName = departmentName;
        }
    }
}
